lexicon/conversion.py

```diff
@@ -119,11 +119,14 @@ def __init__(self, bliss_lexicon, merge_multi_orths_lemmata=False):
                     ** having a synt <=> synt is not None
                 this could lead to INFORMATION LOSS if there are several
                 different synt token sequences in the to-be-merged lemmata
+        :param bool deduplicate_special_lemmata: if True, special lemmata will also be considered
```

Comment:
If you think there are possibly cases where `same_order=True` makes sense, maybe this should not just be a bool but sth like `deduplicate_special_lemmata_type` or `deduplicate_special_lemmata_opts`?